tests/integration-tests/tests/performance_tests/test_scaling.py

```diff
@@ -54,3 +62,211 @@ def test_scaling(
         log_files=["/var/log/parallelcluster/clustermgtd"],
         log_msg=["Found the following bootstrap failure nodes"],
     )
+
+
+def _datetime_to_minute_granularity(dt: datetime):
+    return dt.replace(second=0, microsecond=0)
+
+
+def _get_scaling_time(capacity_time_series: list, timestamps: list, scaling_target: int, start_time: datetime):
+    scaling_target_index = capacity_time_series.index(scaling_target)
+    timestamp_at_full_cluster_size = timestamps[scaling_target_index]
+    scaling_target_time = datetime.datetime.fromtimestamp(
+        float(timestamp_at_full_cluster_size), tz=datetime.timezone.utc
+    )
+    return scaling_target_time, int((scaling_target_time - start_time).total_seconds())
+
+
+SCALING_TARGETS = [1000, 2000, 3000, 4000]
+
+
+@pytest.mark.usefixtures("scheduler")
+@pytest.mark.parametrize(
+    "max_monitoring_time_in_mins, shared_headnode_storage_type, head_node_instance_type, " "scaling_strategy",
```

Comment:
minor: extra space in the parametrize definition

Reply:
fixed